cli/main.go

```diff
@@ -182,13 +182,9 @@ func main() {
 	root.AddCommand(
 		run.Command(ctype),
 		compose.Command(ctype),
+		volume.Command(ctype),
```

Comment:
What happens if you try to use the moby or example type contexts?

Reply:
moby backend just pass parameter to classic CLI
example will get a VolumeService stub